recipe/meta.yaml

```diff
@@ -16,6 +16,8 @@ build:
 
 requirements:
   build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
```

Comment:
This here is exactly the same as the two following lines.

Reply:
You can remove them. This is a bot PR, so you need to do the edits.